### PR TITLE
Scripts for 3279 issue

### PR DESCRIPTION
**Pull request by dboltovskyi**
ATF Test Scripts to check [#3279](https://github.com/SmartDeviceLink/sdl_core/issues/3279)

This PR is **ready** for review.

### Summary
Updates in scripts related to the fix.

### ATF version
develop

### Changelog
 - Corrected `3` existing policy scripts
 - Added `4` new scripts in scope of `HMI PTU` feature
 - Done minor improvements in `common` module of `HMI PTU` feature

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test_scripts/Policies/HMI_PTU/019_No_App_days_trigger.lua [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+runner.Step("Clean environment", common.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", common.start)
+runner.Step("RAI", common.registerApp)
+runner.Step("Activate App", common.activateApp)
+
+runner.Title("Test")
+runner.Step("HMI PTU Successful", common.ptuViaHMI, { updFunc })
+runner.Step("Ignition Off", common.ignitionOff)
+runner.Step("Set SystemDaysAfterEpoch in the past in policy DB", setPtExchangedXDaysAfterEpochInDB)
+runner.Step("New HMI PTU on Days trigger", ignitionOn)
```
Comment (JackLivio) [inline test_scripts/Policies/HMI_PTU/019_No_App_days_trigger.lua]: Experiencing and error on this step. I am on develop for atf and this pr with develop merged into it

```
[19:46:16,984] New_HMI_PTU_on_Ignition_cycles_trigger                                                [FAIL] (11177 ms)
 HMI notification SDL.OnStatusUpdate: Timeout: HMI notification SDL.OnStatusUpdate: Timeout expired
 HMI call BasicCommunication.PolicyUpdate: Timeout: HMI call BasicCommunication.PolicyUpdate: Timeout expired

```

File: test_scripts/Policies/HMI_PTU/020_No_App_ignition_cycles_trigger.lua [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+runner.Step("Activate App", common.activateApp)
+
+runner.Title("Test")
+runner.Step("HMI PTU Successful", common.ptuViaHMI, { updFunc })
+runner.Step("Ignition Off", common.ignitionOff)
+runner.Step("Ignition On without PTU", ignitionOnNoPTU)
+runner.Step("Ignition Off", common.ignitionOff)
+runner.Step("Ignition On without PTU", ignitionOnNoPTU)
+runner.Step("Ignition Off", common.ignitionOff)
+runner.Step("New HMI PTU on Ignition cycles trigger", ignitionOnWithPTU)
```
Comment (JackLivio) [inline test_scripts/Policies/HMI_PTU/020_No_App_ignition_cycles_trigger.lua]: Experiencing and error on this step. I am on develop for atf and this pr with develop merged into it

```
[19:46:16,984] New_HMI_PTU_on_Ignition_cycles_trigger                                                [FAIL] (11177 ms)
 HMI notification SDL.OnStatusUpdate: Timeout: HMI notification SDL.OnStatusUpdate: Timeout expired
 HMI call BasicCommunication.PolicyUpdate: Timeout: HMI call BasicCommunication.PolicyUpdate: Timeout expired

```

File: test_scripts/Policies/HMI_PTU/019_No_App_days_trigger.lua (hunk 1 above)
Comment (dboltovskyi) [reply test_scripts/Policies/HMI_PTU/019_No_App_days_trigger.lua]: @JackLivio Such error should happened for `PROPRIETARY` SDL policy mode. Though script should work fine on `EXTERNAL_PROPRIETARY`.
We suppose this PTU trigger is applicable for both policy modes.
@ShobhitAd Could you please share your thoughts on this?

File: test_scripts/Policies/HMI_PTU/020_No_App_ignition_cycles_trigger.lua (hunk 2 above)
Comment (dboltovskyi) [reply test_scripts/Policies/HMI_PTU/020_No_App_ignition_cycles_trigger.lua]: @JackLivio The same comment as for `019_No_App_days_trigger.lua` script is also applicable here.

File: test_scripts/Policies/HMI_PTU/019_No_App_days_trigger.lua (hunk 1 above)
Comment (JackLivio) [reply test_scripts/Policies/HMI_PTU/019_No_App_days_trigger.lua]: Those triggers should work on both proprietary and external proprietary.

Comment (dboltovskyi) [reply test_scripts/Policies/HMI_PTU/019_No_App_days_trigger.lua]: @JackLivio If so, https://github.com/SmartDeviceLink/sdl_core/issues/3279 issue is still need to be fixed for `PROPRIETARY` for these 2 triggers.

Comment (ShobhitAd) [reply test_scripts/Policies/HMI_PTU/019_No_App_days_trigger.lua]: @dboltovskyi I agree that the PTU should be applicable for both policy modes. Created a fix PR for this issue https://github.com/smartdevicelink/sdl_core/pull/3294.

**Comment by dboltovskyi** [comment]
@ShobhitAd @JackLivio Please notice develop branch has been merged into this PR in order to be aligned with [#3294](https://github.com/smartdevicelink/sdl_core/pull/3294)

**Pull request merged**